extensions/blocks/conversation/edit.js

```diff
@@ -0,0 +1,151 @@
+/**
+ * External dependencies
+ */
+import { filter, map } from 'lodash';
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import {
+	useEffect,
+	useRef,
+	useCallback,
+	useMemo,
+} from '@wordpress/element';
+import {
+	InnerBlocks,
+	InspectorControls,
+	BlockControls,
+} from '@wordpress/block-editor';
+import {
+	Panel,
+	PanelBody,
+	ToggleControl,
+	ToolbarGroup,
+} from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import './editor.scss';
+import ParticipantsDropdown, { ParticipantsSelector } from './components/participants-controls';
+import TranscritptionContext from './components/context';
```

Comment:
Nitpick and non-blocking: One more typo here.
```suggestion
import TranscriptionContext from './components/context';
```